lib/routes/unipd/ilbolive/news.ts

```diff
@@ -0,0 +1,105 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/ilbolive/news',
+    name: 'Il Bo Live - News',
+    url: 'https://ilbolive.unipd.it/it/news',
+    maintainers: ['Gexi0619'],
+    example: '/unipd/ilbolive/news',
+    parameters: {},
+    description: 'Il Bo Live - News',
+    categories: ['university'],
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportRadar: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['ilbolive.unipd.it/it/news'],
+            target: '/unipd/ilbolive/news',
+        },
+    ],
+    handler,
+};
+
+async function handler() {
+    const baseUrl = 'https://ilbolive.unipd.it';
+    const homeUrl = `${baseUrl}/it/news`;
+
+    const response = await got(homeUrl);
+    const $ = load(response.data);
+
+    const items = $('#list-nodes .col')
+        .toArray()
+        .map((el) => {
+            const item = $(el);
+            const title = item.find('.title a').text().trim();
+            const href = item.find('.title a').attr('href');
+            const link = href ? baseUrl + href : null;
+            const category = item.find('.category').text().trim();
+            const image = item.find('.photo img').attr('src');
+            const imageUrl = image ? baseUrl + image : null;
+
+            return title && link
+                ? {
+                      title,
+                      link,
+                      category,
+                      enclosure_url: imageUrl,
+                      enclosure_type: 'image/jpeg',
+                  }
+                : null;
+        })
+        .filter(Boolean);
```

Comment:
```suggestion
    const items = $('#list-nodes .col.-s-6')
        .toArray()
        .map((el) => {
            const item = $(el);
            const title = item.find('.title a').text().trim();
            const href = item.find('.title a').attr('href');
            const link = baseUrl + href;
            const category = item.find('.category').text().trim();
            const image = item.find('.photo img').attr('src');
            const imageUrl = baseUrl + image;

            return {
                title,
                link,
                category,
                enclosure_url: imageUrl,
                enclosure_type: 'image/jpeg',
            };
        });
```
You can use a stricter selector to drop the pagination.